Concepts/concepts_class.md

```diff
@@ -0,0 +1,96 @@
+# Concept Class Type
+
+## Overview
+
+* The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  
+In this way, the class is helpful for data extraction.  This classification details how a concept will be represented 
+(i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when 
+defining concepts, but implementation sites may also create additional custom concept classes for use. 
+
+## Available operations. 
+
+1. [List concept_class types](#list-concept-class-types)
+2. [Create a concept_class_type](#create-a-concept-class-type)
+3. [Update a concept_class type](#update-a-concept-class-type)
+4. [Delete a concept_class type](#delete-a-concept-class-type)
```

Comment:
Remove the term "type(s)"

* List concept classes
* Create a concept class
* Update a concept class
* Delete a concept class